### DISCUSSION TITLE
Is app content body text or fluent text?

### DISCUSSION THREAD

**Discussion by rdoyle720** [Myndex/SAPC-APCA#121] [Q&A] [answered]
I’ve read through several APCA articles and I’m still not clear so I’ll just ask: 

My company creates several web apps. The default font size for these apps is 14px Open Sans. A given screen might contain a lot of text, but it’s buttons, data tables, etc. Rarely would you get a full sentence. 

The only reference I see to 14px regular weight text seems to imply it must hit Lc 90 with it’s background, but I also see references that say Fluent text should be Lc 75 for anything under 18px…help?

**Comment by rdoyle720** [comment]
One of the links here points to https://readtech.org/ARC/, which says for Bronze “There is no specified minimum font size“.  But on   https://www.myndex.com/APCA/ under “ CONFORMANCE: BRONZE LEVELS (preliminary)” there are definite minimums mentioned, ie “ with a font no smaller than 24px/300 weight, 18px/400, 16px/500 and 14px/700”. 

Is one more current than the other?

**Comment by rdoyle720** [comment]
@Myndex

**Comment by Myndex** [comment, marked as answer]
Hi @rdoyle720 

So, because WCAG itself does not cover apps per se, it was not part of the initial guideline, but that is changing.

With ARC are reconsidering the font size issues and focusing more on font weight. This is provided that the app or content respects user settings as they regard to font size.

What we are working on is a reasonable set of minimums in terms of vertical size, and with guidelines for apps.

The main issue is that apps and content respect and/or allow the user to resize fonts, and if they do, the stated minimums will be relaxed. There will still be the guidance regarding "body" "fluent" "sub-fluent" and "spot/fixation" types.


The definition of body-text is expected to change, so that it is not a matter of arbitrary "2.5 lines", but related to actual use/function. 

Also, while it isn't spelled out implicitly, the "Bronze Simple" mode has slightly stiffer thresholds, while the higher level guidelines are more flexible, as the tool generates a sliding scale that is more accurate.

All of these issues are being worked on for the next iteration, which is a little behind schedule at the moment.

As far as the discrepancy in guidelines, that is wrapped up in some political issues in terms of "minimum strictness". That is the one part that is hard to predict—but the "no minimum" is going to be associated with "must respect user font size settings" be they at the OS level or user agent level.


I hope this clarifies and doesn't muddy the waters further. Please continue with these questions as they are helpful as we go though the materials.

**Reply by rdoyle720** [reply]
This helps! Thanks for taking the time.

**Reply by Myndex** [reply]
No problem, we'll have more concrete answers soon, and thank you for the questions as they help us define important issues users/content authors are facing.
